pkg/services/ngalert/notifier/channels_config/plugin.go

```diff
@@ -7,6 +7,7 @@ type NotifierPlugin struct {
 	Heading     string           `json:"heading"`
 	Description string           `json:"description"`
 	Info        string           `json:"info"`
+	Alert       []string         `json:"alert"`
```

Comment:
Im assuming this struct is only ever used for frontend presentation?  (not saved in db or unmarshalled from json)

Reply:
The ngalert NotifierPlugin only appears in channels_config.GetAvailableNotifiers, which is just used to send the list of options to the front-end